src/utils/helpers.ts

```diff
@@ -1,7 +1,7 @@
 import { CommentElementPricing } from "../types";
 import { WideLabel, WideOrgConfig, WideRepoConfig } from "./private";
 
-export const getChainId = (parsedRepo?: WideRepoConfig, parsedOrg?: WideOrgConfig, parsedDefaultConfig?: WideRepoConfig): number => {
+export const getChainId = (parsedRepo: WideRepoConfig | undefined, parsedOrg: WideOrgConfig | undefined, parsedDefaultConfig: WideRepoConfig): number => {
```

Comment:
Use `?` instead of `undefined` as it was before.

Reply:
`?` means optional parameter so if we do that we get an error because the last parameter is not optional.

Reply:
Can you change all of these to destructured parameters (pass in a single object) with the `?` optional parameter markup?

Reply:
You mean like this `{ parsedRepo, parsedOrg,  parsedDefaultConfig }: { parsedRepo?: WideRepoConfig; parsedOrg?: WideOrgConfig; parsedDefaultConfig: WideRepoConfig; }` ?

Reply:
Yes that looks correct!